lib/SimplePie/SimplePie/Item.php

```diff
@@ -2847,9 +2856,9 @@ public function get_enclosures()
 				}
 			}
 
-			if ($enclosure = $this->get_item_tags(SIMPLEPIE_NAMESPACE_RSS_20, 'enclosure'))
+			foreach ((array) $this->get_item_tags(SIMPLEPIE_NAMESPACE_RSS_20, 'enclosure') as $enclosure)
```

Comment:
Fix https://github.com/FreshRSS/FreshRSS/issues/4974

Reply:
can confirm, that it fix it.

Reply:
It breaks the article thumbnails.
Before: the article thumbnails took the first image of article or enclosure.
After: no thumbnail generated from enclosure.

Reply:
Upstream PR https://github.com/simplepie/simplepie/pull/769

Reply:
> It breaks the article thumbnails. Before: the article thumbnails took the first image of article or enclosure. After: no thumbnail generated from enclosure.

@math-GH Please try again ( https://github.com/FreshRSS/FreshRSS/pull/4944/commits/c5c09ea9547faeccd5d44258cb52a2fc2a3cf4ce )

Reply:
Nice. It works fine!